src/main/docker/permissions/caf

```diff
@@ -0,0 +1 @@
+ ALL ALL=(ALL) NOPASSWD: /bin/cp * /etc/pki/trust/anchors*, /usr/sbin/update-ca-certificates
```

Comment:
Is the space at the start of the line necessary?  It just looks strange to me (but I don't know the syntax so keep it if it's normal).